package/environments/base.js

```
@@ -1,16 +1,20 @@
 const { basename, dirname, join, relative, resolve } = require("path")
+const { existsSync, readdirSync } = require("fs")
 const extname = require("path-complete-extname")
-// TODO: Change to `const { WebpackAssetsManifest }` when dropping 'webpack-assets-manifest < 6.0.0' (Node >=20.10.0) support
-const WebpackAssetsManifest = require("webpack-assets-manifest")
-const webpack = require("webpack")
-const rules = require("../rules")
 const config = require("../config")
 const { isProduction } = require("../env")
-const { moduleExists } = require("../utils/helpers")
+
+const pluginsPath = resolve(__dirname, "..", "plugins", `${config.assets_bundler}.js`)
```

ESLint

[error] Replace `__dirname,·"..",·"plugins",·`${config.assets_bundler}.js`` with `⏎··__dirname,⏎··"..",⏎··"plugins",⏎··`${config.assets_bundler}.js`⏎`

(prettier/prettier)